core/core.go

```diff
@@ -169,7 +168,7 @@ func AutocompleteTags(litedb db.LiteDB, query Query) (bool, error) {
 		}
 
 		for _, row := range rows {
-			tag := "#" + row[db.TitleKey]
+			tag := "#" + row[db.TitleKey] + "#"
```

Comment:
Many people (myself one of them) don't use multi-word tags and we want it to behave the same as before. Please check if `len(row[db.TitleKey]) > 1` and only then add the extra `#` to `tag`.